### DISCUSSION TITLE
Too many pop-up ads on the support site

### DISCUSSION THREAD

**Discussion by booni3** [rappasoft/laravel-livewire-tables#1839] [General]
I love the package and use it a lot. The general ads on the site are fine, but the pop-ups are quire infuriating. They seem to pop up on every page as well as multiple times while I am trying to read through a single page. Im not sure if pop-ups provide you with a ton more ad-sense revenue, but if not it might be worth getting rid of at least those ones!

**Comment by lrljoe** [comment]
I have no control over this sadly, but I have been experimenting with hiding ads completely for sponsors, and will chat with @rappasoft about that approach.

**Reply by lrljoe** [reply]
I have however opened a PR against the site, so that the banner ads are removed for Sponsors.

Not massively familiar with the site code though, so it'll need a review before @rappasoft merges it in!

**Comment by grafxflow** [comment]
It would also be nice if the demo/example urls were updated to show the latest versions of tailwind and laravel-livewire-tables as well.

My own thoughts on the adverts are sadly turning me to wanting to move over to another dependancy for livewire tables. It never used to be this bad!

**Comment by rappasoft** [comment]
The ads provide no revenue, i'll remove them as soon as I have time.

**Reply by grafxflow** [reply]
@rappasoft Thanks that would be much appreciated
